packages/hash/frontend/src/theme/components/inputs/MuiOutlinedInputThemeOptions.ts

```diff
@@ -0,0 +1,48 @@
+import { Components, Theme } from "@mui/material";
+
+export const MuiOutlinedInputThemeOptions: Components<Theme>["MuiOutlinedInput"] =
+  {
+    defaultProps: {
+      notched: false,
+    },
+    styleOverrides: {
+      root: ({ theme }) => ({
+        color: "inherit",
+        lineHeight: "18px",
+        borderRadius: "6px",
+        "&:hover": {
+          "& .MuiOutlinedInput-notchedOutline": {
+            borderColor: theme.palette.gray[30],
+          },
+        },
+        "&.MuiOutlinedInput-root": {
+          paddingRight: "unset",
+          paddingLeft: theme.spacing(1.5),
+        },
```

Comment:
Nit: this is equivalent:
```suggestion
        paddingRight: "unset",
        paddingLeft: theme.spacing(1.5),
```